### PR TITLE
Bump pkgs in EC and EPP as per dependerbot

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Bump pkgs in EC and EPP as per dependerbot

**Related issue(s)**

### CODE REVIEW

File: components/event-publisher-proxy/go.mod [hunk 1]
```diff
@@ -3,21 +3,21 @@ module github.com/kyma-project/kyma/components/event-publisher-proxy
...
 require (
-	github.com/cloudevents/sdk-go/v2 v2.13.0
+	github.com/cloudevents/sdk-go/v2 v2.14.0
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/mux v1.8.0
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/kyma-project/kyma/components/application-operator v0.0.0-20221118103320-ffe096ff3455
 	github.com/kyma-project/kyma/components/eventing-controller v0.0.0-20221221134059-d9a237e9ad4f
-	github.com/nats-io/nats-server/v2 v2.9.15
+	github.com/nats-io/nats-server/v2 v2.9.16
```
Comment (marcobebway) [inline components/event-publisher-proxy/go.mod:12]: Should we wait until we actually use this version of NATS?

File: components/eventing-controller/go.mod [hunk 2]
```diff
@@ -4,16 +4,16 @@ go 1.20
...
+	github.com/cloudevents/sdk-go/v2 v2.14.0
 	github.com/go-logr/logr v1.2.4
 	github.com/go-logr/zapr v1.2.3
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/kyma-incubator/api-gateway v0.0.0-20220819093753-296e6704d413
 	github.com/kyma-project/kyma/common/logging v0.0.0-20221118103320-ffe096ff3455
 	github.com/kyma-project/kyma/components/application-operator v0.0.0-20221118103320-ffe096ff3455
 	github.com/mitchellh/hashstructure/v2 v2.0.2
-	github.com/nats-io/nats-server/v2 v2.9.15
+	github.com/nats-io/nats-server/v2 v2.9.16
```
Comment (marcobebway) [inline components/eventing-controller/go.mod:16]: Same https://github.com/kyma-project/kyma/pull/17369/files#r1173675279

File: components/event-publisher-proxy/go.mod (hunk 1 above)
Comment (mfaizanse) [reply components/event-publisher-proxy/go.mod]: reverted

**Review by marcobebway — APPROVED** [review]

**Pull request merged**